programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1573,6 +1571,7 @@ static int run_tunnel(uv_loop_t *ziti_loop, uint32_t tun_ip, uint32_t dns_ip, co
         }
     }
     run_tunneler_loop(ziti_loop);
+    tun->close(tun->handle);
```

Comment:
@smilindave26 the tsdk is now closing the netif_driver when the uv_loop is stopped. I think the apple tunneler will be OK with this, but wanted to point it out.